pkg/backend/backend.go

```diff
@@ -316,6 +316,9 @@ type QueryOperation struct {
 
 // StackConfiguration holds the configuration for a stack and it's associated decrypter.
 type StackConfiguration struct {
+	// List of ESC environments imported by the stack being updated.
+	EnvironmentImports []string
```

Comment:
Should we denote that these are ESC environment imports to differentiate between the environment values below and these

Reply:
oh interesting - how does that work as a single value?

Reply:
synced offline - just confusion on my part about the names / pulumi config being able to be defined in the ESC env and also resolved as part of the CLI